packages/nextjs/next.config.js

```diff
@@ -9,6 +9,18 @@ const nextConfig = {
   eslint: {
     ignoreDuringBuilds: process.env.NEXT_PUBLIC_IGNORE_BUILD_ERROR === "true",
   },
+  serverRuntimeConfig: {
+    providerConfig: {
+      networks: [
+          { name: "mainnet", rpcUrl: "https://eth.drpc.org" },
+          { name: "sepolia", rpcUrl: "https://sepolia.drpc.org", registry: "0x03d5003bf0e79c5f5223588f347eba39afbc3818" }
+      ]
+    },
+    mongoConfig: {
+      connectionStr: "mongodb+srv://verinta:BFWmxukoOsNdIx4x@ceptorclub.rq4oohp.mongodb.net/",
+      DB: "CeptorClub_allan"
```

Comment:
need move to env later

Reply:
I have made a commit to solve this and updated Vercel to include the ENV vars for the deployment,
![image](https://github.com/ceptor-club/blockmagic-tech-team/assets/98447433/4d613e39-b2df-4990-980b-19b251c240ba)